lib/lotus/env.rb

```diff
@@ -0,0 +1,57 @@
+require 'lotus/utils/string'
+
+module Lotus
+  # @since x.x.x
+  # @api private
+  #
+  # @see Lotus.env
+  #
+  # Return environment and check correct environment
+  # Lotus.env.development?
+
+  # @example env = Lotus.env
+  #   # => #<Lotus::Env:0x000000024e56b8 @string="development">
+  #   # env.to_s
+  #     # "development"
+  #
+  #   # env.development?
+  #     # true
+
+  class Env < Utils::String
```

Comment:
naming is hard, what do you think about renaming it to `EnvironmentInqurier` instead? I find `Env` vs `Environment` can cause confusion,